Src/BootCamp.Chapter/AnswerFormatter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class AnswerFormatter
+    {
+        public string GetFormattedAnswerForPersonWithBiggestLoss(PersonAndBalance personWithBiggestLoss)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GetFormattedAnswerForPersonWithBiggestLoss' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)